x-pack/plugins/security_solution/public/common/components/navigation/solution_grouped_nav/solution_grouped_nav.styles.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiThemeComputed } from '@elastic/eui';
+
+const primaryBackground = 'rgba(0, 119, 204, 0.1)';
+
+export const solutionGroupedNavStyles = (euiTheme: EuiThemeComputed<{}>) => ({
+  sideNavItem: `
+    font-weight: 400;
+    &.solutionGroupedNavItem--isPrimary {
+      font-weight: 700;
+    }
```

Comment:
Font weight tokens are found here: https://elastic.github.io/eui/#/theming/typography#font-weight

```suggestion
    font-weight: ${euiTheme.font.weight.regular};
    &.solutionGroupedNavItem--isPrimary {
      font-weight:${euiTheme.font.weight.bold};
    }
```